Integrate overview page (#5027)
PostHog/posthog.com: contents/docs/integrate/client/js/index.mdx

However, you can also leverage the event properties `$set` and `$set_once` to do this via an event.

### Avoid tracking sensitive data

> PostHog puts a great amount of effort into making sure it doesn't capture any sensitive data from your website. (This includes fields like passwords and credit card inputs.) If there are other elements you don't want to be captured, you can add the `ph-no-capture` CSS class.

Reviewer: ```suggestion
PostHog puts a great amount of effort into making sure it doesn't capture any sensitive data from your website (such as fields like passwords and credit card inputs). If there are other elements you don't want to be captured, you can add the `ph-no-capture` CSS class.
```
